### PR TITLE
prep-javascript-for-loops - Kristian Krog

### PR DESCRIPTION
**Pull request by krog-kristian**
prep-js-for-loops

### CODE REVIEW

File: prep-javascript-for-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  console.log(i);
+}
+
+var i = '';
+
+for (i = 0; i < 10; i++) {
+  console.log(i * 2);
+}
+
+for (i = 100; i > 0; i--) {
```
Comment (BrettAlbright) [inline prep-javascript-for-loops/main.js:15]: don't forget to declare the variables `var i = 100`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

File: prep-javascript-for-loops/main.js (hunk 1 above)
Comment (krog-kristian) [reply prep-javascript-for-loops/main.js:15]: updated-js-for-loops

**Pull request merged**